tests/api2/test_initshutdownscript.py

```diff
@@ -0,0 +1,81 @@
+import base64
+import errno
+import os
+import stat
+import uuid
+
+import pytest
+
+from auto_config import ip, ha
+from middlewared.test.integration.utils.client import client
+from middlewared.service_exception import ValidationErrors, ValidationError
+
+TEST_SCRIPT_FILE = '/tmp/.TEST_SCRIPT_FILE'
+_775 = stat.S_IRWXU | stat.S_IRWXG | stat.S_IROTH | stat.S_IXOTH
+
+
+@pytest.fixture(scope='module')
+def ip_to_use():
+    return ip if not ha else os.environ['controller1_ip']
+
+
+@pytest.fixture(scope='module')
+def ws_client(ip_to_use):
```

Comment:
Why is this necessary?

Reply:
So we only establish 1 websocket session for the scope of the module since there are 2 tests being executed (`test_initshutdown_script` and `test_initshutdown_script_bad`)

Reply:
Do you mean why is `ip_to_use` fixture necessary?

Reply:
Should be fixed now.

Reply:
YOu forgot to remove `ip_to_use` argument

Reply:
oops, fixed